packages/documentation-framework/components/example/example.css

```diff
@@ -101,32 +67,32 @@
 }
 
 .pf-v6-c-badge.ws-beta-badge {
```

Comment:
Do you know where this badge is used? I poked around for a minute on the PR preview site and couldn't find this `ws-beta-badge` class.

Reply:
I believe its used to mark props in prop tables as beta like here:
https://patternfly-org-pr-4081-site.surge.sh/components/menus/select#selectlist

Reply:
This looks good but I wonder if we could just remove these styles and use a (compact?) label out of the box. We have a blue/outline label, but also if it's more of an "info" label (formerly blue, updated to purple for v6), maybe that would be a good use case.